change/@ni-nimble-components-35ae0106-c2e5-4744-adc3-d07e2cd543fa.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Since this component is still incubating, all changes can be marked patch. (Should we move it out of incubating sometime? That can be a conversation for another day if you want!)
```suggestion
  "type": "patch",
```

---

I think there are still some items on the checklist before moving it out, and after we do these changes to the API